packages/forms-web-app/src/controllers/lpa-dashboard/decided-appeals.js

```diff
@@ -1,16 +1,26 @@
 const { getLPAUserFromSession } = require('../../services/lpa-user.service');
-
+const { getDecidedAppealsCaseDataV2 } = require('../../lib/appeals-api-wrapper');
+const { mapToLPADecidedData } = require('../../lib/dashboard-functions');
 const {
 	VIEW: {
 		LPA_DASHBOARD: { DECIDED_APPEALS, DASHBOARD }
 	}
 } = require('../../lib/views');
 
 const getDecidedAppeals = async (req, res) => {
+	let appealsCaseData = [];
```

Comment:
Do we need this rather than just `const appealsCaseData = await getDecidedAppealsCaseDataV2(user.lpaCode);` ?

Reply:
This was a legacy declaration, have removed